float32 request_offset
---
bool completed